tests/e2e/cloudcore/cloudcore_suite_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudcore
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
Do not use dot import.

Reply:
I see the case and other codes on the official website do this.
https://github.com/onsi/ginkgo/blob/master/example/books/books_suite_test.go
https://github.com/kubeedge/kubeedge/blob/master/tests/e2e/keadm/keadm_suite_test.go

Reply:
Better to change it :)